packages/manager/modules/octavia-load-balancer/src/octavia-load-balancer.routing.js

```diff
@@ -1,53 +1,13 @@
-import template from './octavia-load-balancer.html';
```

Comment:
I think we can keep the base template here so it will be accessible or all 'octavia-load-balancer' child states. If you move it to octavia-load-balancer.loadbalancers state, i won't be accessible anymore for the create state

Reply:
Done